pymaster/field.py

```diff
@@ -658,3 +669,104 @@ def get_ell_sampling(self):
         """
         ells = lib.get_ell_sampling_flat(self.fl, int(self.fl.fs.n_ell))
         return ells
+
+
+class NmtFieldCatalog(NmtField):
+    """
+    """
+    def __init__(self, positions, weights, field, lmax, lmax_mask=-1,
```

Comment:
It may be more appropriate to use `None` as a default for `lmax_mask`

Reply:
Right, this is now addressed (in both `NmtField` and `NmtFieldCatalog`) with the latest push.